Sources/PointFree/RequestContext.swift

```diff
@@ -42,47 +42,36 @@ public func requireUser<A>(
   -> Middleware<StatusLineOpen, ResponseEnded, A, Data> {
 
     return { conn in
-
-      let currentUser = extractedGitHubUserEnvelope(from: conn.request)
```

Comment:
I rewrote these helpers cause they weren't really working, and `readSessionCookieMiddleware` does all the main heavy lifting.